.circleci/config.yml

```
@@ -171,6 +171,21 @@ commands:
               circleci-agent step halt
             fi
 
+  skip-if-github-only:
+    steps:
+      - run: git fetch upstream
+      - run:
+          name: check for github-only changes
+          command: |
+            MERGE_BASE="$(git merge-base upstream/main HEAD)"
+            DIFF_DIRS="$(git diff-tree --no-commit-id --name-only "$MERGE_BASE" HEAD)"
```

Why do the extra work to find the common ancestor between main and this branch?  Why not just
```suggestion
            DIFF_DIRS="$(git diff-tree --no-commit-id --name-only upstream/main HEAD)"
```
If we're assuming that all PRs are changes from main, then we probably just want to test all of the commits between main and here.

---

/scrolls up
Oh, because that's how the other blocks do it.  Dang it, historical authors. :)